guides/hack/20-attributes/07-predefined-attributes.md

```diff
@@ -191,6 +191,26 @@ The interface type `IMemoizeParam` assists with memorizing objects passed to asy
 - If an exception is thrown, this is not memoized.
```

Comment:
Probably remove this, since it's been replaced by the new section.

This leaves the "Limitations" section with only 1 item, so maybe integrate that into the text above (there's a paragraph about what kinds of argumentsd are allowed where it would probably fit)